app/src/androidTest/java/com/swent/assos/screens/LoginScreen.kt

```diff
@@ -1,7 +1,6 @@
 package com.swent.assos.screens
 
 import androidx.compose.ui.test.SemanticsNodeInteractionsProvider
-import androidx.compose.ui.test.hasTestTag
```

Comment:
You should let this import. It is important for the right execution tests. (Same for every others)